assets/scss/_accordion.scss

```diff
@@ -0,0 +1,37 @@
+//
+// Animation values taken from https://github.com/google/material-design-lite/blob/mdl-1.x/src/_variables.scss
+// Maybe we can bring this into `swarm-constants` or `swarm-animation`?
+//
+
+$A_curve-fast-out-slow-in: cubic-bezier(0.4, 0, 0.2, 1);
+$A_curve-default: $A_curve-fast-out-slow-in;
+$A_duration: 0.2s;
+
+.accordionPanel-animator {
+	overflow: hidden;
+	transition: height $A_duration $A_curve-default;
+}
+
```

Comment:
these can be added to swarm-animation (which has its own animation constants). I don't want to get in the way of this getting merged. we need to think of a way to get this project to pull in animation.scss from swarm-animate pretty soon.